src/cryptsetup/cryptsetup-generator.c

```diff
@@ -487,6 +524,32 @@ static crypto_device *validate_uuid_and_get_crypto_device(const char *uuid, cons
         return d;
 }
 
+static char *get_uuid_candidate(const char *value, const char *key, size_t *ret_pos, int *ret_error)
+{
```

Comment:
coding style: bracket belongs on same line as function name, and ret_error should be the return value, as above

Reply:
Removed in updated PR